### ISSUE TITLE
NSQ plugin failed with version 1.0.0-compat

### ISSUE THREAD

**Issue by r0bj** [influxdata/telegraf#2672]
### Relevant telegraf.conf:

```
[[outputs.influxdb]]
  urls = ["http://prod.os-metrics-db.service.sjc.consul:8086"] # required
  database = "telegraf"
  retention_policy = ""
  write_consistency = "any"
  timeout = "5s"

[[inputs.nsq]]
  endpoints = ["http://localhost:4151"]
```

### System info:

telegraf: v1.2.1
NSQ: 1.0.0-compat
OS: ubuntu 14.04

### Steps to reproduce:

```
# telegraf --config telegraf.conf --test
* Plugin: inputs.nsq, Collection 1
> nsq_server,server_version=,host=nsq-r1,server_host=localhost:4151 server_count=0i,topic_count=0i 1492158440000000000
```

### Expected behavior:

Something similar to output when using NSQ 0.3.8:
https://gist.github.com/r0bj/d5804619992c6fb885106cbe9dbb30f2

### Actual behavior:

```
# telegraf --config telegraf.conf --test
* Plugin: inputs.nsq, Collection 1
> nsq_server,server_version=,host=nsq-r1,server_host=localhost:4151 server_count=0i,topic_count=0i 1492158440000000000
```

No data is returned, version string is empty.

**Comment by freeseacher** [comment]
any news on that ? willing to update nsqd but cant do it without monitoring

**Comment by abdullin** [comment]
Hitting the same problem. Is there any estimate on when there will be next update on this bug?

**Comment by danielnelson** [comment]
Fixed in #3148
